text/0148-sampling-probability.md

```diff
@@ -0,0 +1,794 @@
+output adjusted count in this case is the output from weighted
+sampling divided by the (absolute) point value.
+
+#### Zero adjusted count
+
+An adjusted count with zero value carries meaningful information,
+specifically that the item participated in a probabilistic sampling
+scheme and was not selected.  A zero value can be be useful to record
+events outside of a sample, when they provide useful information
+despite their effective count.  We can use this to record error
+exemplars, for example, even when they are not selected by the
+Sampler.
+
+## Proposed Tracing specification
+
+For the standard OpenTelemetry Span Sampler implementations to support
+a range of probability sampling schemes, this document recommends the
+use of a Span attribute named `sampling.adjusted_count` to encode an
+unbiased adjusted count computed by a Sampler reflecting the whole 
+population of spans.
+
+The value the `sampling.adjusted_count` attribute under this proposal
+MUST be an unbiased estimate of the total population count represented
+by the individual event.
+
+### Suggested text
+
+After
+[Sampler](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#sampler)
+and before [Builtin
+Samplers](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#built-in-samplers),
+a new section will introduce the term "adjusted count" and relate it
+to inclusion probability.  For example:
+
+```markdown
+### Probability sampling
+
+Probability Samplers are Samplers that output statistically unbiased
+inclusion probability.  Inclusion probability in the context of
+tracing is the *effective* probability of the Sampler returning
+`RECORD_AND_SAMPLE` when invoked, which can be decided locally or 
+derived from the context when the [W3C Trace Context is-sampled
+flag](https://www.w3.org/TR/trace-context/#sampled-flag) is in use.
+
+#### Adjusted count span attribute
+
+The recommended way to convey inclusion probability *for events*
+sampled in OpenTelemetry is in the form of an **adjusted count**,
+which is the reciprocal (i.e., mathematical inverse function) of
+inclusion probability.
+
+The implied goal of probability sampling is to support estimating the
+count of spans in the population using the spans that were sampled.
+The adjusted count associated with a Span is the expected value of the
+number of identical Spans within the population that each Span
+represents.  Probability samplers SHOULD ensure the samples they
+compute are unbiased, which implies that the sum
+of adjusted counts in the sample approximates the true count of spans in the
+population.
+
+The adjusted count in an unbiased probability sampling scheme SHOULD
+be expressed using a Span attribute named `sampling.adjusted_count`
+when it represents the whole population of events.  Adjusted count 
+attributes MAY be integer or floating-point values.
+
+#### Inclusion probability tracestate value
```

Comment:
I am not sure it is clear how the SDKs should behave given that the propagation is optional. Propagating probability is a clear additional overhead that some sites may not want to pay, especially if they don't use any inflationary or partial sampling. So how does an SDK know if it should or should not include the probability in the tracestate? Should it be a configuration of the SDK (tricky because guaranteeing identical configuration throughout all microservices is quite difficult)?

Reply:
The text is outdated, resolving. I am now proposing non-optional support. See https://github.com/open-telemetry/oteps/pull/168